src/features/encoders/avif/client/index.tsx

```diff
@@ -228,7 +236,7 @@ export class Options extends Component<Props, State> {
                         <div class={style.optionOneCell}>
                           <Range
                             min="0"
-                            max="63"
+                            max="99"
```

Comment:
Change `"99"` to `{MAX_QUALITY - 1} // MAX_QUALITY would mean lossless`?

See line 194.